helm/fiaas-skipper/templates/deployment.yaml

```diff
@@ -36,6 +36,7 @@ spec:
           protocol: TCP
         livenessProbe:
           initialDelaySeconds: 10
+          timeoutSeconds:15
```

Comment:
This endpoint is just returning 200 OK without actually doing anything. Isn't 15 seconds a bit excessive for the timeout?

Also, I think you need a space between the key and the value